include/picongpu/logging.hpp

```diff
@@ -19,21 +19,12 @@
 
 #pragma once
 
-#include "picongpu/simulation_defines.hpp"
-
 #include "picongpu/debug/PIConGPUVerbose.hpp"
 
-#include <pmacc/Environment.hpp>
-#include <pmacc/mappings/kernel/AreaMapping.hpp>
-#include <pmacc/math/Vector.hpp>
-#include <pmacc/types.hpp>
-
-
 namespace picongpu
 {
-    using namespace pmacc;
-
     // short name for access verbose types of picongpu
     using picLog = PIConGPUVerbose;
 
+    using pmacc::log;
```

Comment:
why does this single line solves all issues with the removed includes above?

Reply:
this single line is pulling only a single function from pmacc into the picongpu namespace.
This avoids that I need to write everywhere `pmacc::log`

This is much better than `using namespace pmacc;`